scripts/lighthouse/logResults.js

```diff
@@ -0,0 +1,42 @@
+const chalk = require('chalk'); // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
Maybe have a `/* eslint-disable-rule no-console */` at the top of this file, instead of overwriting each individual instance.